### DISCUSSION TITLE
filedldialog ?

### DISCUSSION THREAD

**Discussion by rraammiinn** [TomSchimansky/CustomTkinter#1008] [Q&A] [answered]
does it have a custom filedldialog too ?
it's most necessary thing for python gui

**Comment by Akascape** [comment, marked as answer]
@rraammiinn Obviously not. You have to use the tkinter filedialogs. 
In short, all tkinter things can be implemented with customtkinter.

**Reply by avalon60** [reply]
Yes, it's a bit of a shame that the tkinter.filedialog doesn't appear to let you change text, background colours etc. At lease it's not obvious if there is. Unless there's something hidden in the depths of the documentation somewhere.

**Comment by rraammiinn** [comment]
thanks

**Comment by rraammiinn** [comment]
I'm using eel for a gui app but for filedldialog I have to use tkinter
filedldialog
I tried plyer filechooser but it didn't work
